internal/operator/config/config.go

```diff
@@ -13,12 +13,14 @@
 
 package config
 
-var (
+const (
 	DefaultContourImage           = "docker.io/projectcontour/contour:main"
 	DefaultEnvoyImage             = "docker.io/envoyproxy/envoy:v1.17.0"
 	DefaultMetricsAddr            = ":8080"
```

Comment:
I know this isn't part of this change but `8080` isn't the default metrics port. This would then conflict with the default http port.

Reply:
@stevesloka `DefaultMetricsAddr` is the default metrics address for the operator (not Contour's metrics address).

Reply:
Ahh you're right, got those mixed up. 🙃